src/Nancy/HttpLinkBuilder.cs

```diff
@@ -0,0 +1,49 @@
+namespace Nancy
+{
+    using System.Collections.Generic;
+    using System.Linq;
+
+    /// <summary>
+    /// Builds an RFC 5899 Link HTTP header as a <see cref="List{T}" /> of <see cref="HttpLink" /> objects.
+    /// </summary>
+    /// <seealso cref="HttpLink" />
+    public class HttpLinkBuilder : List<HttpLink>
```

Comment:
True, but I don't know of an interface that is a middle-ground between `IList<T>` and `IEnumerable<T>`. While the other methods and properties of `List<T>` aren't currently in use by Nancy itself, who knows what consumers might want to do with the `HttpLinkBuilder`? They might find the indexer useful, for instance. If we shouldn't cater to thought use cases, I can of course instead implement `IEnumerable<T>` and add an interface-free `Add()` method, if you think that's cleaner?